src/content/docs/tutorial-optimize-telemetry/optimize-your-ingest.mdx

```diff
@@ -0,0 +1,1537 @@
+---
+title: "Optimize your data ingest"
+metaDescription: "Taking your ingested and reported ingest data and optimizing it."
+redirects:
+  - /docs/telemetry-data-platform/manage-data/data-governance
+  - /docs/data-governance
+  - /docs/telemetry-data-platform/get-started/manage-data/data-governance
+  - /telemetry-data-platform/get-started/manage-data/data-governance
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing
+---
+
+Setting up your ingest data and charting it into digestable charts is useful for any organization. But using that data to optimize your telemetry should be the ultimate goal, and is what we cover in the last part of this tutorial series. By setting up observability objectives, creating your optimization plan, and using data reduction techniques to carry that plan out, you can reduce your redundant ingest data to reduce your costs and increase the value of the data you're ingesting. 
```

Comment:
This intro can either be cut or refreshed. It feels very ephemeral and abstract and I'm wondering if we can make this more concrete for the intended audience.

Reply:
I'll take another crack at it, thank you!